sdk/go/common/workspace/plugins_test.go

```diff
@@ -740,6 +740,63 @@ func TestPluginDownload(t *testing.T) {
 		assert.Equal(t, int(l), len(readBytes))
 		assert.Equal(t, expectedBytes, readBytes)
 	})
+	t.Run("GitHub Releases with disallowed", func(t *testing.T) {
+		t.Setenv("GITHUB_TOKEN", token)
+		version := semver.MustParse("5.32.0")
+		spec := PluginSpec{
+			PluginDownloadURL: "",
+			Name:              "mockdl",
+			Version:           &version,
+			Kind:              apitype.PluginKind("resource"),
+		}
+		source, err := spec.GetSource()
+		require.NoError(t, err)
+		attempts := 0
+		getHTTPResponse := func(req *http.Request) (io.ReadCloser, int64, error) {
+			attempts++
+
+			fmt.Println("req.Header.Get(\"Authorization\")", req.Header.Get("Authorization"))
```

Comment:
There's a print here and below, are these intentionally left in?

Reply:
Doh, no I was just debugging and forgot these.